features/support/github.js

```diff
@@ -142,6 +150,107 @@ function createPullRequest(branch, repoOwner, repoName) {
     });
 }
 
+/**
+ * Creates a lightweight tag.
+ * @method createTag
+ * @param  {String}   tag               Name of the tag
+ * @param  {String}   branch            The branch to create the file in
+ * @param  {String}   [repoOwner]       Owner of the repository
+ * @param  {String}   [repoName]        Name of the repository
+ * @return {Promise}
+ */
+function createTag(tag, branch, repoOwner, repoName) {
+    const owner = repoOwner || 'screwdriver-cd-test';
+    const repo = repoName || 'functional-git';
+
+    return octokit.git.getRef({
+        owner,
+        repo,
+        ref: `heads/${branch}`
+    })
+        .then((referenceData) => {
+            const sha = referenceData.data.object.sha;
+
+            return octokit.git.createRef({
+                owner,
+                repo,
+                ref: `refs/tags/${tag}`,
+                sha
+            });
+        })
+        .catch((err) => {
+            Assert.strictEqual(err.status, 422);
+        });
```

Comment:
I think this indent is too deep.
If this indent is an ESLint rule, there is no problem.